edi/views/edi_exchange_type_views.xml

```diff
@@ -35,6 +35,15 @@
                             <field name="exchange_file_auto_generate" />
                             <field name="ack_type_id" />
                         </group>
+                        <group name="manual" group="base.group_no_one">
```

Comment:
when #275  is merged we can use the specific group for advanced settings I added there :wink:

Reply:
OK